### ISSUE TITLE
Image naming conventions

### ISSUE THREAD

**Issue by jackhkmatthews** [thisissoon/angular-image-loader#8]
The use of `image`, `images`, `ResponsiveImage`, `RetinaImage`, `placeholder` and `fullback` can be confusing.

Perhaps the use of simply `image` and `images` should be replaced with a more strict use of `ResponsiveImage`, `placeholder`, `fallback`, `RetinaImages`and `RetinaImage`.

Or a rethink of naming conventions should be carried out.

**Comment by edoparearyee** [comment]
yeah i agree it can be confusing, open to suggestions on better naming
